resources/assets/coffee/react/beatmap-discussions/post.coffee

```diff
@@ -113,7 +113,9 @@ export class Post extends React.PureComponent
             div
               className: "#{bn}__user-badge"
               if userBadge?
-                div className: "user-group-badge user-group-badge--#{userBadge}"
+                div
+                  className: "user-group-badge user-group-badge--#{userBadge}"
+                  title: osu.trans "beatmap_discussions.user_title.#{userBadge}" if !@props.user.is_bot
```

Comment:
why no title for bot?

Reply:
i thought its self explanatory already

Reply:
If there's going to be a tooltip there, it should be for all of them

Reply:
to be honest, i think not having a tooltip for bot is reasonable, it's a special case and also tooltip would need to say same as badge what's quite dumb